ppr-ui/src/components/mhrTransfers/HomeOwnersGroupError.vue

```diff
@@ -22,12 +22,9 @@
   </span>
   <!-- Transfer Sale or Gift error messages -->
   <span v-else-if="isTransferDueToSaleOrGift && TransSaleOrGift.hasMixedOwnersInGroup(groupId)">
-    <span v-if="hasOneHomeOwnerGroup">
-      {{ transfersErrors.hasMixedOwnerTypes }}
-    </span>
-    <span v-else>
-      {{ transfersErrors.hasMixedOwnerTypesInGroup }}
-    </span>
+              {{ hasOneHomeOwnerGroup ?
+                MixedRolesErrors.hasMixedOwnerTypes :
+                MixedRolesErrors.hasMixedOwnerTypesInGroup }}
   </span>
```

Comment:
Spacing seems* off here, i could be wrong but looks to have excess indentation, forgive me if not the case.